### PR TITLE
CLN: removed pandas.sandbox

### PR DESCRIPTION
**Pull request by gfyoung**
Deprecated back in `0.16.0` <a href="https://github.com/pydata/pandas/pull/9615">here</a>, and the module seems to be rarely, if at all, used according to @jorisvandenbossche .

### CODE REVIEW

File: doc/source/faq.rst [hunk 1]
```diff
@@ -110,78 +110,6 @@ details.
 Visualizing Data in Qt applications
```
Comment (sinhrks) [inline doc/source/faq.rst]: How about removing whole section, and moves to ecosystem?

Comment (gfyoung) [inline doc/source/faq.rst]: So remove the entire thing about Qt applications?  I don't fully understand the second part of your question.

Comment (sinhrks) [inline doc/source/faq.rst]: Yes. The second part means ecosystem.rst which lists some related projects.

- https://github.com/pydata/pandas/blob/master/doc/source/ecosystem.rst

Comment (gfyoung) [inline doc/source/faq.rst]: Got it.  Done.

**Comment by gfyoung** [comment]
@sinhrks : made the requested doc comments (no need to run Travis tests again hence the `[ci skip]`), so ready to merge if there are no other concerns.

**Comment by jreback** [comment]
thank you sir!

**Pull request closed without merging**